src/map/map.ts

```diff
@@ -447,6 +447,14 @@ export class PropertiesMap {
                 }
                 const min = axis.min.value;
                 const max = axis.max.value;
+                if (min > max) {
+                    sendWarning(
+                        `${name}: inserted min and max values are such that min > max! Values are resetted to the last ones inserted.`
```

Comment:
```suggestion
                `Inserted min value is greater than max value for the ${name}!
                         Values are resetted to the last ones inserted.`
```
I would change the warnign message to this so it is more alligned with other error messages and also is not on a single long line in the source code file.